content/backstage/plugins/datadog.md

```diff
@@ -0,0 +1,153 @@
+
+      const ServiceEntityPage = ({ entity }: { entity: Entity }) => (
+        &lt;EntityPageLayout>
+            &lt;EntityPageLayout.Content
+                path="/datadog/*"
+                title="datadog"
+                element={<DatadogRouter entity={entity} />}
+                />
+        &lt;/EntityPageLayout>
+        );
+
+  - intro: Add widget to your Overview tab.
+    language: typescript
+    code: |
+      // packages/app/src/plugins.ts
+        const OverviewContent = ({ entity }: { entity: Entity }) => (
+            &lt;Grid container spacing={3} alignItems="stretch">
+            ...
+            {isDatadogWidgetAvailable(entity) && (
+            &lt;Grid item>
+                &lt;DatadogGraphWidget entity={entity} />
+            &lt;/Grid>
+            )}
+            ...
+        &lt;/Grid>
+        );
+        
+
+---
+
+## How to embed a datadog dashboard in Backstage
+
+### Obtain the dashboard URL from Datadog that you will need for your metadata. 
+
+* Login to your Datadog account.
+
+### Get the dashboard URL.
+
+* Navigate to the dashboards list by hovering over dashboards on the page's left-hand side and selecting the dashboard list.
+
+* Select a dashboard from this list.
+
+* Within the dashboard you have chosen, click the settings cog on the screen's right-hand side, circled in red.
+
+![dashboard](../../assets/dd-dashboard.png)
+
+
+* Copy the URL from the Sharing textbox.
+
+* This URL is the value you need for the `datadoghq.com/dashboard-url` annotation.
+
+![dashboard share](../../assets/dd-dashboard-share.png)
+
+
+### Adding the annotations and the values from Datadog to your component's metadata file.
+
+```yaml
+apiVersion: backstage.io/v1alpha1
+kind: Component
+metadata:
+  name: sample-service
+  description: |
+    A sample service
+  annotations:
+    datadoghq.com/dashboard-url: <<DATADOGURL>>
+```
+
+## How to embed a datadog graph in Backstage
+
+* Login to your Datadog account.
+
+### Get the graph token.
+
+* Click on the graph pencil, circled in red, from your dashboard.
+
+![dashboard](../../assets/dd-dashboard-2.png)
+
+* Click on the Share tab, choose a timeframe, graph size and legend. Click generate the embedded code. 
+
+* Copy the token value that is highlighted in the red square.
+
+* this token is the value you need for the `datadoghq.com/graph-token` annotation
+
+![dashboard](../../assets/dd-graph-share.png)
+
+### Adding the annotations and the values from Datadog to your component's metadata file.
+
+```yaml
+apiVersion: backstage.io/v1alpha1
+kind: Component
+metadata:
+  name: sample-service
+  description: |
+    A sample service
+  annotations:
+    datadoghq.com/graph-token: <<TOKEN>
+```
+
+## Security
+
+A word of note regarding the security of the datadog dashboards and graphs.
+
+The instructions provided for sharing dashboards and graphs generate a URL.
+
+This URL is public to anyone who bears it.
+
+If obtained by another actor, it is usable by them.
+
```

Comment:
The phrasing is a little awkward in the security section. How about something like this?

> Sharing Datadog dashboards and graphs makes them public on the internet and accessible by anyone with the URL. Make sure you do not share these URLs with untrusted actors.

Reply:
Updated as your text came across better.